### PR TITLE
Polio 1177 link afro map to country view

### PR DESCRIPTION
**Pull request by quang-le**
Explain what problem this PR is resolving

Related JIRA tickets :POLIO-1177, POLIO-1178

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [x] Are my typescript files well typed
- [x] New translations have been added or updated if new strings have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

- Add a pop up with a link to the LQAS page of the campaign linked to the shape. The link is to the base page, i.e: the round selection is not mirrored. This is because the Afro map filters campaigns and round displayed based on dates, so the same map can display an mix of round numbers. making each map aware of the round selected for each country in the other map would require a significant (and tricky) rewrite of the Afro map

## How to test

Explain how to test your PR.
- Go to afro map
- click a country, test the link
- Do the same with district view

## Print screen / video

https://github.com/BLSQ/iaso/assets/38907762/9387796e-99e8-44ad-af78-d6954eaccd7a

## Notes

Has already merged https://github.com/BLSQ/iaso/pull/810
Has already merged with  https://github.com/BLSQ/iaso/pull/812
is merging into polio refactor https://github.com/BLSQ/iaso/pull/811

### CODE REVIEW

File: hat/assets/js/apps/Iaso/domains/tasks/types.ts
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/tasks/types.ts:1]: 🤟

File: hat/assets/js/apps/Iaso/hooks/taskMonitor.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                return false;
+            },
+            onSuccess: data => {
+                if (!data) {
+                    setEnabled(false);
+                    if (invalidateQueries.length > 0) {
+                        invalidateQueries.forEach(queryKey =>
+                            queryClient.invalidateQueries(queryKey),
+                        );
+                    }
```
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/hooks/taskMonitor.ts]: for this you can pass directly 'invalidateQueries' to `useSnackQuery`, it accepts an array of string now

File: plugins/polio/js/src/domains/LQAS-IM/LQAS/LqasAfroOverview/Map/LqasAfroPopUp.tsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        view === COUNTRY ? shape.data?.country_name : shape.data?.district_name;
+
+    // District view needs more space for French translation
+    const labelSize = view === COUNTRY ? 4 : 6;
+    const valueSize = view === COUNTRY ? 8 : 6;
+    const countryId = view === COUNTRY ? shape.id : shape.country_id;
+
+    return (
+        // ignore classname TS error // @ts-ignore
+        // @ts-ignore
```
Comment (beygorghor) [inline plugins/polio/js/src/domains/LQAS-IM/LQAS/LqasAfroOverview/Map/LqasAfroPopUp.tsx]: This one needs to be fixed one day

**Review by beygorghor — APPROVED** [review]

File: plugins/polio/js/src/domains/LQAS-IM/LQAS/LqasAfroOverview/Map/LqasAfroPopUp.tsx (hunk 2 above)
Comment (quang-le) [reply plugins/polio/js/src/domains/LQAS-IM/LQAS/LqasAfroOverview/Map/LqasAfroPopUp.tsx]: yeah, so annoying

**Pull request merged**